components/threads/threads.app.mjs

```diff
@@ -32,24 +32,25 @@ export default {
     blocks: {
       label: "Blocks",
       type: "string[]",
-      description: "Add one or more blocks to your thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`",
+      description:
+        "Add one or more blocks to a thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`",
```

Comment:
```suggestion
        "Add one or more blocks to a thread. You can use [Markdown](https://www.markdownguide.org/basic-syntax/) to format text. Try `# This is a block`"
```

Reply:
Changed "Try" to "try" to follow sentence case rule from Google's Developer docs style guide.

Reply:
This would be grammatically incorrect as sentences start with capitals.